logsearch-platform-deployment.yml

```diff
@@ -0,0 +1,21 @@
+director_uuid: (( param "Please set the UUID of your BOSH Director" ))
+
+update:
+  serial: false
+  canaries: 1
+  canary_watch_time: 30000-600000
+  update_watch_time: 5000-600000
+  max_in_flight: 1
+  max_errors: 1
+
+stemcells:
+- alias: default
+  name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+  version: latest
+
+name: logsearch-platform
+
+releases:
+- {name: logsearch, version: latest}
+- {name: logsearch-for-cloudfoundry, version: latest}
```

Comment:
Yeah, because it ships with logstash filters and kibana dashboards that should theoretically work with cf platform logs.